uportal-war/src/main/java/org/jasig/portal/groups/pags/dao/jpa/PersonAttributesGroupTestDefinitionImpl.java

```diff
@@ -46,14 +51,14 @@
  * @author Shawn Connolly, sconnolly@unicon.net
  */
 @Entity
-@Table(name = "UP_PAGS_TEST")
+@Table(name = JpaPersonAttributesGroupTestDefinitionDao.TABLENAME_PREFIX)
```

Comment:
FWIW I would probably have the prefix UP_PAGS and use it for group, member, test, etc., but this works too.